ow_power_system/src/power_system_node.cpp

```diff
@@ -128,6 +153,7 @@ void PowerSystemNode::Run()
 
   // Finally subscribe to mechanical power topic (Watts)
   m_mechanical_power_sub = m_nh.subscribe("/mechanical_power/average", 1, &PowerSystemNode::powerCallback, this);
+  m_power_fault_sub = m_nh.subscribe("/faults/power_faults", 1, &PowerSystemNode::powerFaultsCallback, this);
```

Comment:
I'm noting the queue size of 1 here, vs 1000 in other subscriptions.  I don't know what the best choices are, but curious if there's a motivation for these values.

Reply:
Here I'm keeping the same value as what the mech power sub is using in order to make sure it's using the most updated values from faults.